Bottom: f5dd75d8422f3f1047ee8f650efda3f55407c00b
Top:    8a27ed8c35d90e7954f56a2067efd3b82de67225
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-05-27 01:03:00 -0400

Refresh of mpc5200-ac97

---

diff --git a/sound/soc/fsl/mpc5200_psc_ac97.c b/sound/soc/fsl/mpc5200_psc_ac97.c
index 4342b51..794a247 100644
--- a/sound/soc/fsl/mpc5200_psc_ac97.c
+++ b/sound/soc/fsl/mpc5200_psc_ac97.c
@@ -35,7 +35,7 @@ static unsigned short psc_ac97_read(struct snd_ac97 *ac97, unsigned short reg)
 	unsigned int val;
 
 	/* Wait for command send status zero = ready */
-	status= spin_event_timeout(!(in_be16(&psc_dma->psc_regs->sr_csr.status) &
+	status = spin_event_timeout(!(in_be16(&psc_dma->psc_regs->sr_csr.status) &
 				MPC52xx_PSC_SR_CMDSEND), 100, 0);
 	if (status == 0) {
 		pr_err("timeout on ac97 bus (rdy)\n");
@@ -97,7 +97,7 @@ static void psc_ac97_cold_reset(struct snd_ac97 *ac97)
 	out_8(&regs->op1, MPC52xx_PSC_OP_RES);
 	udelay(10);
 	out_8(&regs->op0, MPC52xx_PSC_OP_RES);
-	udelat(50);
+	udelay(50);
 	psc_ac97_warm_reset(ac97);
 }
